vsphere_plugin_common/__init__.py

```diff
@@ -2937,24 +2937,41 @@ def delete_file(self, datacenter_name, datastorepath):
         self.si.content.fileManager.DeleteFile(datastorepath, dc.obj)
 
     def upload_file(self, datacenter_name, allowed_datastores,
-                    remote_file, data, host, port):
+                    allowed_datastore_ids, remote_file, data, host,
+                    port):
         dc = self._get_obj_by_name(vim.Datacenter, datacenter_name)
         if not dc:
             raise NonRecoverableError(
                 "Unable to get datacenter: {datacenter}"
                 .format(datacenter=repr(datacenter_name)))
+        ctx.logger.debug(
+            "Will be checked storages with such ids: {ids}, and names: {names}"
```

Comment:
"Will check storage with IDs: {ids}; and names: {names}" might be a better way of phrasing this.